x/ccv/provider/keeper/grpc_query.go

```diff
@@ -110,7 +110,10 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 	}
 
 	clientID, _ := k.GetConsumerClientId(ctx, consumerId)
-	topN := k.GetTopN(ctx, consumerId)
+	powerShapingParameters, err := k.GetConsumerPowerShapingParameters(ctx, consumerId)
```

Comment:
`GetConsumerPowerShapingParameters` returns an error when params aren't set for the chain.
The error shouldn't be checked here.

Reply:
I modified the CreateConsumer method to always set power shaping params (if not provided by the user, then it will just set the defaults -- zero values).